sdks/python/apache_beam/runners/portability/portable_stager_test.py

```diff
@@ -56,7 +56,7 @@ def _stage_files(self, files):
           describing the name of the artifacts in local temp folder and desired
           name in staging location.
     """
-    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+    server = grpc.server(CollapsingThreadPoolExecutor(max_workers=10))
```

Comment:
Consider omitting the argument here (and for other tests, aside from artifact_serving_test where we want to control the parallelism).